app/src/main/res/layout/recent_tabs_header.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="56dp"
+    android:layout_marginTop="40dp">
```

Comment:
This looks wrong. We're adding an inconsistent amount of margin above this section:

![out](https://user-images.githubusercontent.com/1370580/121587031-59301200-ca02-11eb-99bf-87255715e04d.png)

Reply:
isn't the pager dots?

Reply:
No, you can see in the left screenshot which "container" the top sites dots goes in. Also, removing this margin line takes away the weird space. I added 10dp in this screenshot:

We want to use something like 8/16/24dp.
<img src=https://user-images.githubusercontent.com/1370580/121588798-56ceb780-ca04-11eb-8e76-f2ac5628aa19.png width=280 />

Reply:
Still unsure about this. Let's discuss and follow up.